.github/workflows/refresh-doc.yml

```diff
@@ -0,0 +1,75 @@
+name: refresh-doc
+on:
+  push:
+    branches:
+      - hannah-github-actions
+
+env:
+  WEB_URL: 'https://cold-pigs-fail-174-94-59-90.loca.lt' # This needs to be updated to the live web server!
+
+jobs:
+  build:
+    runs-on: macos-latest
```

Comment:
Is macOS required? Can this also run on a Linux machine?

Reply:
Will update this to 'ubuntu-latest'